config/mochaConfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "reporterEnabled": "mocha-junit-reporter, xunit, spec",
+  "xunitReporterOptions": {
+    "output": "xunit.xml"
+  },
+  "mochaJunitReporterReporterOptions": {
+    "mochaFile": "junit-custom.xml"
+  }
+}
```

Comment:
Top level mochaConfig that is needed for anything running mocha directly. This is necessary for the non-vscode packages that do not run VSCode's test bin which run mocha programmatically.